packages/gdk_pixbuf.rb

```diff
@@ -31,17 +17,17 @@ class Gdk_pixbuf < Package
   def self.build
     system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX} --with-x11"
     system "make"
+    # generate cache
+    system "mkdir -p #{CREW_LIB_PREFIX}/gdk-pixbuf-2.0/2.10.0/"
+    system "sudo #{CREW_DEST_PREFIX}/bin/gdk-pixbuf-query-loaders --update-cache"
```

Comment:
Have you tried without `sudo`?  I think it should work since that directory is writable by `chronos` user.